src/schema/server_settings.py

```diff
@@ -1,22 +1,50 @@
 import re
+from enum import Enum
 from typing import List, Optional
 
 from pydantic import BaseModel, Field
 from steamship import SteamshipError
 
+from schema.quest import QuestDescription
 from schema.stable_diffusion_theme import StableDiffusionTheme
 
 
-def validate_prompt_args(prompt: str, valid_args: List[str]):
+def validate_prompt_args(
+    prompt: str, valid_args: List[str], prompt_name: str
+) -> Optional[str]:
     regex = r"\{(.*?)\}"
     matches = re.finditer(regex, prompt)
     variable_names = sorted({match.group(1) for match in matches})
+    missing_vars = []
     for variable_name in variable_names:
         if variable_name not in valid_args:
-            return False
-    return True
+            missing_vars.append(variable_name)
+
+    if len(missing_vars) > 0:
+        return f"{prompt_name} uses the following variable names which are not available: [{' '.join(missing_vars)}]"
+    else:
+        return None
+
+
+class AvailableVoice(str, Enum):
```

Comment:
thought: it'd be really nice if there was someway to get this intended functionality to be in-sync with any changes to `_SUPPORTED_ELEVEN_VOICES`.

if these are just for const-like behavior, I wonder if we could make something like:

```python
AVAILABLE_VOICES = list(_SUPPORTED_ELEVEN_VOICES.keys())
```

Reply:
Very open to less-duplicative ways to write that.  Just slapped it down the first way that came to mind.